[2021-09-07 05:43:21,865] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: json_etl_dag.extract_data 2021-09-06T00:00:00+00:00 [queued]>
[2021-09-07 05:43:21,881] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: json_etl_dag.extract_data 2021-09-06T00:00:00+00:00 [queued]>
[2021-09-07 05:43:21,881] {taskinstance.py:1094} INFO - 
--------------------------------------------------------------------------------
[2021-09-07 05:43:21,882] {taskinstance.py:1095} INFO - Starting attempt 1 of 1
[2021-09-07 05:43:21,882] {taskinstance.py:1096} INFO - 
--------------------------------------------------------------------------------
[2021-09-07 05:43:21,893] {taskinstance.py:1114} INFO - Executing <Task(PythonOperator): extract_data> on 2021-09-06T00:00:00+00:00
[2021-09-07 05:43:21,896] {standard_task_runner.py:52} INFO - Started process 28500 to run task
[2021-09-07 05:43:21,899] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'json_etl_dag', 'extract_data', '2021-09-06T00:00:00+00:00', '--job-id', '201', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/etl.py', '--cfg-path', '/tmp/tmp1l2ixy2o', '--error-file', '/tmp/tmp8klef4fk']
[2021-09-07 05:43:21,900] {standard_task_runner.py:77} INFO - Job 201: Subtask extract_data
[2021-09-07 05:43:21,931] {logging_mixin.py:109} INFO - Running <TaskInstance: json_etl_dag.extract_data 2021-09-06T00:00:00+00:00 [running]> on host 731700afc5ff
[2021-09-07 05:43:22,045] {taskinstance.py:1251} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=json_etl_dag
AIRFLOW_CTX_TASK_ID=extract_data
AIRFLOW_CTX_EXECUTION_DATE=2021-09-06T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-09-06T00:00:00+00:00
[2021-09-07 05:43:22,260] {python.py:151} INFO - Done. Returned value was: None
[2021-09-07 05:43:22,270] {taskinstance.py:1211} INFO - Marking task as SUCCESS. dag_id=json_etl_dag, task_id=extract_data, execution_date=20210906T000000, start_date=20210907T054321, end_date=20210907T054322
[2021-09-07 05:43:22,318] {local_task_job.py:151} INFO - Task exited with return code 0
[2021-09-07 05:43:22,350] {local_task_job.py:261} INFO - 1 downstream tasks scheduled from follow-on schedule check
